static/scss/_button.scss

```diff
@@ -171,8 +171,15 @@
   font-size: 15px;
   font-weight: 400;
   cursor: pointer;
+  background: none;
+  border: none;
+  padding: 3px 0;
 }
 
+.mm-minor-action:hover {
+  text-decoration: underline;
+  }
```

Comment:
can you fix the indentation here? this curly brace should be aligned with the left margin.